grails-app/taglib/org/pih/warehouse/AuthTagLib.groovy

```diff
@@ -88,6 +88,12 @@ class AuthTagLib {
             out << body()
     }
 
+    def ifUserHasRoles = {attrs, body ->
```

Comment:
Please for now follow here the convention of `isUserInRole` and make this one `isUserInAllRoles` instead `ifUserHasRoles`